src/Spreadsheet.js

```diff
@@ -252,11 +252,12 @@ class Spreadsheet<CellType, Value> extends PureComponent<{|
     this.root = root;
   };
 
+  getCellComponent = memoizeOne(enhanceCell);
```

Comment:
I don't under this.

Reply:
It's memoizing the application of `enhanceCell` to the `Cell`  prop. If `Cell` changes between renders, the new component will be enhanced and that enhanced component will be returned from `getCellComponent` every time it gets called (unless `Cell` changes again)

In a functional component we'd be able to use hooks like so:

```js
function Spreadsheet({ Cell, ... }) {
  const ConnectedCell = useMemo(() => enhanceCell(Cell), [Cell]);
  // now we can use ConnectedCell
}
```

If that makes it a bit clearer.

Reply:
Great. Can you add a short doc comment to explain this

Reply:
Will do :)